src/main/java/com/booleanuk/Poker.java

```diff
@@ -28,4 +79,12 @@ public String extensionThreeMethods() {
         // You will also need to add the relevant tests to Extension3Test.java
         return "Replace this method with your own methods and tests";
     }
+
+    public static void main(String[] args){
+        Poker poker = new Poker();
+
+        System.out.println(poker.map);
+    }
+
+
 }
```

Comment:
Your work on the `winningPair` method shows a clear understanding of evaluating pairs, effectively using a hashmap for card values. For full marks have a go at ext 1 and 2. Good work.